src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/ApprovalStep.tsx

```diff
@@ -35,16 +36,16 @@ export const ApprovalStep = ({
   hopExecutionState,
   isLastStep,
   isLoading,
+  onError,
```

Comment:
If we want to follow the `on/handle` convention 
```suggestion
  onError: handleError,
```